components/pedigree/resources/src/main/resources/pedigree/view/studySelectionDialog.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * The UI Element for displaying prompts to the user in a movable/semi-transparent dialogue for study selection
+ *
+ * @class OkCancelDialogue
+ */
+define([], function(){
+  var StudySelectionDialog = Class.create( {
+
+      initialize: function(studies) {
+          var _this = this;
+
+          this.studies = studies;
+
+          var mainDiv = new Element('div', {'class' : "study-selection-dialog"});
+          var container = new Element('div', {'class' : 'xform'});
+
+          mainDiv.insert(container);
+
+          var submitButton = new Element('input', {type : 'button', value : "Submit", 'class' : 'button'});
+          mainDiv.insert(submitButton);
+          submitButton.observe('click', function(event) {
+              selectedStudies = $$('.study-selection-dialog input[type="radio"].study-input');
+              var studyName = "";
+              selectedStudies.each( function(item) {
+                  if (item.checked) {
+                      studyName = item.value;
+                  }
+              });
+              _this.submit(studyName);
+              _this.hide();
+          });
+
+          var cancelButton = new Element('input', {type : 'button', value : "Cancel", 'class' : 'button secondary'});
```

Comment:
a minor point, but study selection dialogue pops up either from the node menu, or from patient legend, both of which use different styles for the buttons, I think we should use uniform button styles within the same workflow